lib/pageflow/global_config_api.rb

```diff
@@ -44,17 +44,11 @@ def configure(&block)
     # @return [Configuration]
     # @since 0.9
     def config_for(target)
-      config = build_config(
-        target.respond_to?(:entry_type) && target.entry_type.name
-      ) do |c|
-        c.enable_features(target.enabled_feature_names(c))
-      end
-
-      if target.respond_to?(:entry_type)
-        config = Configuration::ConfigView.new(config, target.entry_type)
+      if target.is_a?(EntryTemplate)
+        config_for(Account.find(target.account_id))
+      else
+        config_for_account_or_entry(target)
       end
-
-      config
```

Comment:
This does not use the entry template's entry type yet to scope, for example, widget types by entry type. If we want to save that for a later step, instead of doing this change we could also do `config_for(entry_template.account)` in the [one place where it is used](https://github.com/codevise/pageflow/pull/1420/files#diff-b378d3e9e492d24e8547d4d5fc204542R2). Otherwise it needs a spec.

Reply:
To me it seems better to drop this for now, and do `config_for(entry_template.account)`